docs/zh/connector-v2/sink/Hive.md

```diff
@@ -0,0 +1,522 @@
+# Hive
+
+> Hive 数据接收器
+
+## 支持的版本
+
+已经验证的版本 :
+- 2.3.9
+- 3.1.1
+
+# 使用依赖
+
+当你使用Spark/Flink时, 你需要保证已经与Hive进行了集成.  
+当你需要使用Zeta引擎时, 你需要将这些依赖放到`$SEATUNNEL_HOME/lib/`目录中.
+- `seatunnel-hadoop3-3.1.4-uber.jar`
+- `hive-exec-<hive_version>.jar`
+- `libfb303-0.9.3.jar`
+- `hive-jdbc-<hive_version>.jar` (当设置了`hive_jdbc_url`参数, 需要使用`savemode`功能时)
+
+## 主要特性
+
+- [x] [精准一次](../../concept/connector-v2-features.md)
+  我们使用二阶段提交来保证精准一次
+
+- [x] 支持的文件类型
+
+  - [x] text
+  - [x] csv
+  - [x] parquet
+  - [x] orc
+  - [x] json
+- [x] 支持的压缩方式
+  - [x] none (default)
+  - [x] lzo
+  - [x] snappy
+  - [x] lz4
+  - [x] gzip
+  - [x] brotli
+  - [x] zstd
+
+## 数据类型映射
+
+| Hive Data Type | SeaTunnel Data Type |
+|----------------|---------------------|
+| tinyint        | byte                |
+| smallint       | short               |
+| int            | int                 |
+| bigint         | long                |
+| float          | float               |
+| double         | double              |
+| decimal        | decimal             |
+| timestamp      | local_date_time     |
+| date           | local_date          |
+| string         | string              |
+| varchar        | string              |
+| boolean        | boolean             |
+| binary         | byte array          |
+| arrays         | array               |
+| maps           | map                 |
+| structs        | seatunnel row       |
+| char           | not supported       |
+| interval       | not supported       |
+| union          | not supported       |
+
+## 连接器选项
+
+|              名称               |   类型    |       是否必要       |             默认值              |                                                                        描述                                                                        |
+|-------------------------------|---------|------------------|------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------|
+| table_name                    | string  | yes              | -                            | 模板表名称 例如: `db1.table1`, 如果source端支持多表, 你可以使用 `${database_name}.${table_name}` 来生成表名,  `${database_name}`和`${table_name}` 这两个值会根据source端生成的信息进行替换 |
```

Comment:
No binding the source, any sink can use these variables

Reply:
Done